openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -306,14 +306,12 @@ This endpoint is used by a Credential Issuer that is already interacting with an
 
 ## Credential Offer {#credential-offer}
 
-The Credential Issuer sends Credential Offer using an HTTP GET request or an HTTP redirect to the Wallet's Credential Offer Endpoint defined in (#client-metadata).
+The Credential Issuer passes the Credential Offer to the Wallet, by value or by reference, using HTTP GET request or an HTTP redirect to the Wallet's Credential Offer Endpoint defined in (#client-metadata). 
```

Comment:
```suggestion
The Credential Issuer passes the Credential Offer to the Wallet, by value or by reference (#client-metadata). 
```
The Credential Offer may be send by using custom URL schemes, so I would remove this condition.